tests/unit/notification.test.ts

```
@@ -0,0 +1,98 @@
+import { describe, it, expect, beforeEach } from "vitest";
```

ESLint

[error] Resolve error: File './.nuxt/tsconfig.json' not found.
    at je (/home/jailuser/git/node_modules/get-tsconfig/dist/index.cjs:7:6227)
    at ie (/home/jailuser/git/node_modules/get-tsconfig/dist/index.cjs:7:7177)
    at se (/home/jailuser/git/node_modules/get-tsconfig/dist/index.cjs:7:7892)
    at Object.$e [as getTsconfig] (/home/jailuser/git/node_modules/get-tsconfig/dist/index.cjs:7:7997)
    at /home/jailuser/git/node_modules/eslint-import-resolver-typescript/lib/index.cjs:257:36
    at Array.map (<anonymous>)
    at initMappers (/home/jailuser/git/node_modules/eslint-import-resolver-typescript/lib/index.cjs:251:26)
    at Object.resolve (/home/jailuser/git/node_modules/eslint-import-resolver-typescript/lib/index.cjs:123:3)
    at withResolver (/home/jailuser/git/node_modules/eslint-module-utils/resolve.js:121:23)
    at fullResolve (/home/jailuser/git/node_modules/eslint-module-utils/resolve.js:142:22)

(import/namespace)